tools/topology/sof/pipe-amp-ref-capture.m4

```diff
@@ -106,5 +106,5 @@ indir(`define', concat(`PIPELINE_PCM_', PIPELINE_ID), Echo Reference Capture PCM
 
 
 # PCM capabilities supported by FW
-PCM_CAPABILITIES(Echo Reference Capture PCM_ID, `S32_LE,S24_LE,S16_LE', 48000, 48000, 2, PIPELINE_CHANNELS, 2, 16, 192, 16384, 65536, 65536)
+PCM_CAPABILITIES(Echo Reference Capture PCM_ID, CAPABILITY_FORMAT_NAME(PIPELINE_FORMAT), 48000, 48000, 2, PIPELINE_CHANNELS, 2, 16, 192, 16384, 65536, 65536)
```

Comment:
@keyonjie for my education, what is the pcm_converter?

Reply:
And what problem does this help fix?

Reply:
@ranj063 pcm_converter is a converter help to convert format to the same one of the dai format added inside the firmware. With it, the volume/pga component is not mandatory when the dai component is configured to different format with others (e.g. host).

This PR will help fix issue where we are claiming the wrong supported formats in the PCM capability, e.g. our host buffer is allocated as S16_LE so we can support S16_LE only, but with today's code it is `S32_LE,S24_LE,S16_LE' in the pcm capability, so end user can run it with S24_LE or S32_LE and eventually end up to FW internal errors. See bug like this for detail: https://github.com/thesofproject/sof/issues/3318

Reply:
@keyonjie care to reflect this explanation in the commit message?

Reply:
@dbaluta sure. Adding the pcm capability section but not the pcm_converter related one which actually is not changed in this PR.